ACC2024/RobotDynamics/@RigidBody2D/RigidBody2D.m

```diff
@@ -46,9 +46,15 @@
             end
 
         end
-
+        
+        % [x,y,theta]
         function position = forwardKinematics(obj, pointInBodyFrame)
             position = obj.CoordinateFrame.HomogeneousFromBase * [pointInBodyFrame.ColumnVector; 1];
+            position = [position; obj.CoordinateFrame.RotationFromBase.Theta];
+        end
+        
+        function angle = getCOMAngle(obj)
```

Comment:
COM is a point and it does not have an angle. You can call it get rigid body angle.